smoke-test/src/test/java/org/opennms/smoketest/BSMAdminIT.java

```diff
@@ -637,6 +687,34 @@ public void testCanEditPersistedEdge() throws InterruptedException {
         bsmAdminPage.delete(serviceName);
     }
 
+    @Test
+    public void testCanEditTransientAttribute() throws InterruptedException {
+        // create Business Service with one Attribute
+        final String serviceName = createUniqueBusinessServiceName();
+        BsmAdminPageEditWindow bsmAdminPageEditWindow = bsmAdminPage.openNewDialog(serviceName).addAttribute("foo", "bar");
+        wait.until(pageContainsText("foo=bar"));
+
+        // cancel
+        bsmAdminPageEditWindow.cancel();
```

Comment:
Please verify afterwards that the attribute `foo = bar` has not been set and please delete the Business Service afterwards.